### ISSUE TITLE
PKI X.509 Cert / Signature verifier

### ISSUE THREAD

**Issue by pinkforest** [RustCrypto/rustls-rustcrypto#47]
Should we have one as well ? :sweat_smile: 

It would be a lot of work tho - webpki still has dependency to ring

**Comment by tarcieri** [comment]
There is work underway on a certificate verifier based on `x509-cert`. Note that certificate verification is the electric third rail of the X.509 ecosystem, and difficult to implement in a manner that conforms to other implementations.

See: https://github.com/RustCrypto/formats/issues/838

**Comment by pinkforest** [comment]
Yeah cool think that is catered already somewhere else and we don't really need it here so closing
